plugins-scripts/Classes/F5/F5BIGIP/Component/CpuSubsystem.pm

```diff
@@ -28,7 +28,7 @@ sub overall_init {
 sub init {
   my $self = shift;
   $self->get_snmp_tables('F5-BIGIP-SYSTEM-MIB', [
-      ['cpus', 'sysCpuTable', 'Classes::F5::F5BIGIP::Component::CpuSubsystem::Cpu'],
+      ['cpus', 'sysCpuSensorTable', 'Classes::F5::F5BIGIP::Component::CpuSubsystem::Cpu'],
```

Comment:
This should be carefully reviewed. I have only F5 with Firmware 11.X so I don't know what will happen with older devices.
Not sure if it is worth or needed to support both MIBs at the same time.